### DISCUSSION TITLE
[A-L] (9 Settembre 2021) Esercizio 7

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#325] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152792360-3900346b-809c-49fa-ae87-0059d29e2245.png)

**Comment by matypist** [comment]
Esercizio e soluzioni analoghe a #120 ?

**Comment by LuginiAndrea** [comment]
L'errore consiste nel prendere per vero che $\forall n \in \mathbf{N}, n + 1 = a + b$, con $0 \leq a, b \leq n$. 
Consideriamo $n = 0 \implies 1 = a + b$, con $0 \leq a, b \leq 0$. Ne segue quindi che stiamo dicendo che $1 = 0 + 0$, impossibile
